README.md

```diff
@@ -68,6 +69,18 @@ You can jump directly to "*Available apps, installation procedure*"
 
 Ansible
 -------------------------
+### Coreos bootstrap
+Before running the cluster playbook you must satisfy the following requirements:
+* On each CoreOS nodes a writable directory **/opt/bin** (~400M disk space)
+
+* Uncomment the variable **ansible_python_interpreter** in the file `inventory/group_vars/all.yml`
+
+* run the Python bootstrap playbook
+```
+ansible-playbook -u smana -e ansible_ssh_user=smana  -b --become-user=root -i inventory/inventory.cfg coreos-bootstrap.yml
```

Comment:
I think this step can be done automaticaly in the main cluster.yml  with a condition "when: ansible_os == CoreOS"